db/migrate/20241112161111_remove_microservice_v2_tables.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RemoveMicroserviceV2Tables < ActiveRecord::Migration[7.2]
+  def change
+    drop_table :microservice_registrations
```

Comment:
This will probably cause hard crashes when the migration has been run but AWS hasn't switched the traffic over yet. Maybe save this for a follow-up PR?

Reply:
makes sense. removed!